src/main/java/com/minecolonies/colony/Colony.java

```diff
@@ -759,7 +759,7 @@ private boolean areAllColonyChunksLoaded(@NotNull TickEvent.WorldTickEvent event
 
     private void cleanUpBuildings(@NotNull TickEvent.WorldTickEvent event)
     {
-        @Nullable List<AbstractBuilding> removedBuildings = null;
+        @Nullable final List<AbstractBuilding> removedBuildings = new ArrayList<>();
```

Comment:
Why make this change? It simplifies the logic, but it requires us to make an ArrayList every time this method is called. Which with most calls wouldn't need to happen.